test/videojs-http-streaming.test.js

```diff
@@ -2889,15 +2894,68 @@ QUnit.test('configures eme if present on selectedinitialmedia', function(assert)
   this.player.tech_.hls.masterPlaylistController_.trigger('selectedinitialmedia');
 
   assert.deepEqual(this.player.eme.options, {
-    previousSetting: 1,
+    previousSetting: 1
+  }, 'did not modify plugin options');
+
+  assert.deepEqual(this.player.currentSource(), {
+    src: 'manifest/master.mpd',
+    type: 'application/dash+xml',
     keySystems: {
       keySystem1: {
         url: 'url1',
         audioContentType: 'audio/mp4; codecs="audio-codec"',
-        videoContentType: 'video/mp4; codecs="video-codec"'
+        videoContentType: 'video/mp4; codecs="video-codec"',
+        pssh: 'test'
+      }
+    }
+  }, 'set source eme options');
+});
+
+QUnit.test('ignores configures eme if not provided by source', function(assert) {
```

Comment:
minor: should probably read something like "does not set source keySystems if keySystems not provided by source"